contracts/BondedSortitionPool.sol

```diff
@@ -140,11 +140,11 @@ contract BondedSortitionPool is AbstractSortitionPool {
             ownerAddress
         );
 
-        if (eligibleStake < poolParams.minimumStake) { return 0; }
+        if (eligibleStake < poolParams.minimumStake) {return 0;}
 
-        // Weight = floor(eligibleStake / mimimumStake)
+        // Weight = floor(eligibleStake / poolWeightDivisor)
         // Ethereum uint256 division performs implicit floor
-        // If eligibleStake < minimumStake, return 0 = ineligible.
+        // If eligibleStake < poolWeightDivisor, return 0 = ineligible.
```

Comment:
Might make more sense to move this comment above the minimum stake check a few lines up

Reply:
👍 please see https://github.com/keep-network/sortition-pools/pull/78/commits/fb66e85a75057da00242f3a3d92c1e5126960820